### PR TITLE
Requirements changed.

### PR DESCRIPTION
**Pull request by xmnlab**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,29 +12,16 @@ source:
...
-    - ipywidgets
-    - ipython
   run:
     - python
-    - numpy >=1.12
-    - scikit-learn
-    - matplotlib
-    - jupyter
-    - ipywidgets
-    - ipython
```
Comment (ocefpaf) [inline recipe/meta.yaml:37]: So these are not needed or are they optional?

**Comment by ocefpaf** [comment]
@xmnlab now that this is in a feedstock it is fair game to self merge. Everything looks OK BTW.

File: recipe/meta.yaml (hunk 1 above)
Comment (xmnlab) [reply recipe/meta.yaml:37]: @ocefpaf these packages are needed ... 

But in setup.py I don't import this packages*, should I keep these packages in requirements?

* https://github.com/conda-forge/staged-recipes/pull/3247#discussion_r126273270)

Comment (ocefpaf) [reply recipe/meta.yaml:37]: These are `run` time, so yes. Only the section above (`build`) can be simplified.

Comment (xmnlab) [reply recipe/meta.yaml:37]: oh i see .. I will make the changes :)

**Pull request merged**